pages/common/pytest.md

```diff
@@ -0,0 +1,28 @@
+# pytest
+
+> Run Python tests.
+> More information: <https://docs.pytest.org/>.
+
+- Run tests from a specific file:
+
+`pytest {{path/to/test_file.py}}`
```

Comment:
```suggestion
- Run tests from specific files:

`pytest {{path/to/test_file1.py path/to/test_file2.py ...}}`
```

Optional change: several files possibly can be passed to conform command option [specification](https://manned.org/pytest.1): `python -m pytest [...]`.